sdk/javascript/test/BaseValue_spec.js

```diff
@@ -0,0 +1,123 @@
+const { BaseValue } = require('../src/BaseValue');
+const converter = require('../src/utils/converter');
+const { expect } = require('chai');
+
+describe.only('BaseValue', () => {
+	const DEFAULT_VALUE = BigInt('0x456789ABCDEF0123');
+
+	class MockValue extends BaseValue {
+		constructor(value = 0) {
+			super(MockValue.SIZE, value);
+		}
+
+		static deserialize(arrayInput) {
+			return new MockValue(converter.intFromBytes(arrayInput, MockValue.SIZE, false));
+		}
+	}
+	MockValue.SIZE = 8;
+
+	const canCreateBaseValueFactory = isSigned => (value, size, expectedValue) => {
+		// Act:
+		const val = new BaseValue(size, value, isSigned);
```

Comment:
val/value =? value/rawValue ?

Reply:
this mimics py, where field name == .value
(and in py tests there's actually val used and `val.value`)